frontend/js/router.js

```diff
@@ -14,6 +14,9 @@ class Router {
     this.unregistered_urls = ["/", "/home", "/login", "/register", "/email_verification"];
     console.log("Router: constructor called");
 
+    const formData = new FormData();
+    formData.append("refresh", "");
+    const r = api.post_multipart("/token/refresh/", formData).then(async (result) => {
```

Comment:
Would not it make more sense to call the refresh endpoint after you got the "Logged-in" status? If that one is false, you call the token-refresh. And if that one fails, too, you log out the user?